serialization/publish.gradle

```diff
@@ -0,0 +1,52 @@
+apply plugin: 'com.novoda.bintray-release'
+apply plugin: 'maven'
+
+if (project.getGradle().startParameter.taskNames.any{it.contains('bintrayUpload')}) {
+    group = "com.twitter.serial"
+    version = project.VERSION_NAME
+
+    gradle.taskGraph.whenReady { taskGraph ->
+        taskGraph.getAllTasks().find{it.path == ":$project.name:generatePomFileForMavenPublication"}.doLast {
+            file("build/publications/maven/pom-default.xml").delete()
+            pom {
+                project {
+                    name "Serial"
+                    artifactId "serial"
+                    packaging project.name == 'compiler' ? 'jar' : 'aar'
```

Comment:
I think we can just use `packaging jar`.

Reply:
Based on our POM file here, https://repo1.maven.org/maven2/com/twitter/serial/serial/0.1.5/serial-0.1.5.pom, I think it's actually "packaging 'aar'" that we want. Agree that we don't need this ternary though

Reply:
I think JAR is the correct value since the artifact is a jar file.